sdk/communication/communication-email/swagger/README.md

```diff
@@ -38,3 +38,13 @@ directive:
     transform: >
       $["x-ms-client-name"] = "bcc"
 ```
+
+### Remove "To" from the required properties
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.EmailRecipients
+    transform: >
+      $["required"] = []
```

Comment:
Why are we adding this here and not fixing the swagger itself?

Reply:
The swagger is currently in the process of being edited, this is just a temporary fix.